### PR TITLE
N cameras

### PR DESCRIPTION
**Pull request by ethanrublee**

### CODE REVIEW

**Review by isherman — APPROVED** [review]

File: cpp/farm_ng/calibration/apriltag_rig_calibrator.cpp [hunk 1]
```diff
@@ -179,10 +192,16 @@ void ModelError(ApriltagRigModel& model) {
                    -1);
       }
     }
+    cv::imshow("reprojection", image);
```
Comment (ethanrublee) [inline cpp/farm_ng/calibration/apriltag_rig_calibrator.cpp:195]: TODO remove.

File: docker/cpp.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,63 @@
+FROM ubuntu:18.04
```
Comment (ethanrublee) [inline docker/cpp.Dockerfile:1]: I guess these should be removed, bad merge? @isherman

Comment (isherman) [reply docker/cpp.Dockerfile]: done

File: cpp/farm_ng/capture_video_dataset.cpp [hunk 3]
```diff
@@ -137,6 +137,7 @@ class CaptureVideoDatasetProgram {
   }
 
   void on_event(const EventPb& event) {
+    // TODO(isherman): Discuss right filter with @ethanrublee
```
Comment (isherman) [inline cpp/farm_ng/capture_video_dataset.cpp:140]: @ethanrublee what's the right filter to use here? should it read the `camera.json` configuration?

Comment (isherman) [reply cpp/farm_ng/capture_video_dataset.cpp:140]: - rely on camera.json
- calculate per-camera count
- search for other "tracking_camera/front" strings

Comment (isherman) [reply cpp/farm_ng/capture_video_dataset.cpp]: See 1089c3aea97530613a3dd4aad4a77209bd7d8132

**Pull request merged**